omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+
+* Metabolomics: The study of the set of small molecules in a cell known as **metabolites** that result from cellular metabolism.
+
+* Microbiomics: The study of a system's microorganisms, such as those that live in the human digestive system.
+
+
+### Multi-omics
+
+In "Multi-omics approaches to disease", Hasin et al. write:
+
+>Each type of omics data, on its own, typically provides a list of differences associated with the disease. These data can be useful both as markers of the disease process and to give insight as to which biological pathways or processes are different between the disease and control groups.
+
+All of these biology domains that we have talked about so far are interconnected, and so it is perhaps not very surprising that there are circumstances in which it makes sense to take a **multi-omic** approach for a particular study. Biological processes are complex and can rarely be explained by just one set of molecules. Perhaps a disease-associated genomic variant might be due to contributions of differential regulation (the realm of transcriptomics) and/or changes in protein expression (which might be elucidated by proteomic analysis). A study into a specific disease might wish to examine all of the various biological processes that are altered, from cellular metabolism to gene expression. There are a variety of ways in which multiple omics domains might contribute to a more thorough understanding of biological processes!
+
+### Quiz: Omics topics
+
+Which of the following omics topics involves studying gene expression by quantifying or sequencing RNA?
+
+[( )] Genomics
+[(X)] Transcriptomics
+[( )] Proteomics
+[( )] Metabolomics
+***
+<div class = "answer">
+Transcriptomics is the study of the RNA transcripts in a system, which is considered one measure of gene expression under certain conditions. Genomics is the study of the genome, which is DNA; proteomics deals with proteins, and metabolomics studies the small molecules resulting from cell metabolism.
+</div>
+***
+
+## Caveats and Challenges
+
+While omics approaches can be powerful, this doesn't mean that an omics approach is always the "best", nor that omics studies are guaranteed to provide every answer to every question! There are several caveats to be aware of when planning or reading about an omics study:
+
+* Heterogeneity of populations: According to Hasin et al., "[i]nsights gained from omics approaches are mostly comparative" (2017), meaning that omics studies involve comparing groups to each other (such as "disease vs. healthy" or "treatment vs control", for example). Inherent heterogeneity in any of these populations makes it difficult to distinguish important differences, or "signal", from unrelated variation, also called "noise." Because of the amount of data being analyzed in omics studies, this problem can be amplified compared to traditional approaches.
```

Comment:
```suggestion
* **Heterogeneity of populations:** According to Hasin et al., "[i]nsights gained from omics approaches are mostly comparative" (2017), meaning that omics studies involve comparing groups to each other (such as "disease vs. healthy" or "treatment vs control", for example). Inherent heterogeneity in any of these populations makes it difficult to distinguish important differences, or "signal", from unrelated variation, also called "noise." Because of the amount of data being analyzed in omics studies, this problem can be amplified compared to traditional approaches.
```